docs/quality-assurance/principles-of-safety-assurance.md

```diff
@@ -0,0 +1,23 @@
+# What is safety ?
+
+So the first question is what is safety in relation to UKHO products:
+
+* An error in a product that could potentially impact the safety of the mariner
+* A delay to the mariner that could potentially impact the safety of the mariner
+
+# Some general principles
+
+* Reliance on manual processes is minimised
+* Best practices in software and test engineering are applied
+* Sound human engineering principles are applied to the design of the software use interface to minmise the probability of human error
+* Software with safety critical functionality must have clear evidance that all safety requirements have been met or relevant 
+  mitigations have ben agreed.
+* Failure modes, including hardware, software, human and system are addressed in the design of the software.
+* Speak up, if you feel that safety is being compromised in some way, raise it!!!!!.
+* Exercise judgement, not the same as going out on a limb but exercising common sense
+* ALARP – As low as reasonably practicable
```

Comment:
I'm not sure about this.  What do we mean by reasonable exactly?  I think we need to say somewhere who defines what is reasonable.  Is it the PO?